{"name": "Address中provice不输入新增失败", "status": "passed", "attachments": [{"name": "stdout", "source": "a1a6e772-e608-4229-999f-a627d2f4c0aa-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "Parameter", "value": "{'case_no': '75', 'casename': 'Address中provice不输入新增失败', 'mode': '地址管理', 'data': '{\"fistname\":\"fistname\",\"lastname\":\"lastname\",\"contactnumber\":\"contactnumber\",\"coordinated_area1\":\"\",\"coordinated_area2\":\"coordinated_area2\",\"coordinated_area3\":\"C2\",\"coordinated_postalCode\":\"coordinated_postalCode\",\"coordinated_addressLine\":\"coordinated_addressLine\"}', 'assert_way': 'class name=ant-form-explain', 'result': 'Please input province!'}"}], "start": 1651220668370, "stop": 1651220684402, "uuid": "82eeb3a2-67da-464e-afa1-e0d33b780045", "historyId": "9594056537da39009f8a141db53989a9", "testCaseId": "840f25e4c9ed223ada744fce7d35f48a", "fullName": "test_main.Test_Main#test_address", "labels": [{"name": "feature", "value": "地址管理"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "Test_Main"}, {"name": "host", "value": "DESKTOP-NB3FDE0"}, {"name": "thread", "value": "14900-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_main"}]}